### ISSUE TITLE
Can i run subsocial without Polkadot

### ISSUE THREAD

**Issue by alinayani** [dappforce/subsocial-web-app#439]
Can i use subsocial without polkadot ? if yes how

also can Subsocial actions be signed automatically without an extension ? if yes how

**Comment by alinayani** [comment]
can anyone reply on this please

**Comment by grumbly180** [comment]
Just get polkadot.

**Comment by alinayani** [comment]
> Just get polkadot.

Didn't Get your point. can you please elaborate a bit ?

**Comment by grumbly180** [comment]
The easiest way to use subsocial is with a polkadot.js wallet. https://polkadot.js.org/

If you aren't sure how to use one just message back.

**Comment by alinayani** [comment]
Yeah that's what subsocial web app is providing default. But I am asking for a way to use something other than polkadot to sign transactions.

**Comment by grumbly180** [comment]
let me amend my first response.

No, just use polkadot
